ministryofjustice/correspondence_tool_staff
# CT-3125 a few changes related to he scope of stats figure for cabinet office report

## Description

This PR is to fix the bugs being found by Laura for cabinet office reports around the following figur
1 - C section 
<img width="689" alt="Screenshot 2020-10-12 at 14 52 36" src="https://user-images.githubusercontent.com/26111671/95754353-cbe1c580-0c9a-11eb-9229-4a32949238ee.png">

2 - Outcome section 
<img width="679" alt="Screenshot 2020-10-12 at 14 52 43" src="https://user-images.githubusercontent.com/26111671/95754342-c7b5a800-0c9a-11eb-974b-51f6876c5735.png">

3 - Exemption part
<img width="672" alt="Screenshot 2020-10-12 at 14 53 11" src="https://user-images.githubusercontent.com/26111671/95754313-c1273080-0c9a-11eb-8ae0-4c9b2248e0d8.png">

## Self-review checklist


### Screenshots


### Related JIRA tickets

https://dsdmoj.atlassian.net/secure/RapidBoard.jspa?rapidView=25&modal=detail&selectedIssue=CT-3125
### Deployment


### Manual testing instructions

1 - Login as Branston use
2 - Click "Reports" from top menu
3 - Click "Download report" from "Cabinet office report" for FOI
4 - Check the figure
